packages/rehype-mathjax/lib/core.js

```diff
@@ -2,19 +2,25 @@ const visit = require('unist-util-visit')
 
 module.exports = createPlugin
 
-function createPlugin(displayName, createRenderer) {
+function createPlugin(displayName, createRenderer, chtml = false) {
```

Comment:
```suggestion
function createPlugin(displayName, createRenderer, chtml) {
```

Default of undefined is fine as well and supported everywhere